File: test/fixtures/testsuites/failing_submission_test.rb
require 'minitest/autorun'

require File.expand_path('../fake_submission.rb', __FILE__)

class FailingSubmissionTest < Minitest::Unit::TestCase

  def test_this_will_pass
    @failing_submission = FailingSubmission.new
    assert @failing_submission.this_is_false
  end
  
end

File: test/fixtures/testsuites/failing_submission_test.exs
Code.load_file("test/fixtures/testsuites/failing_submission.exs")
ExUnit.start

defmodule FailingSubmissionTest do
  use ExUnit.Case, async: true
  doctest FailingSubmission

  test "this fails" do
    assert FailingSubmission.this_is_false == true
  end
  
end